codersdk/users.go

```diff
@@ -309,7 +309,7 @@ func (c *Client) DeleteUser(ctx context.Context, id uuid.UUID) error {
 		return err
 	}
 	defer res.Body.Close()
-	if res.StatusCode != http.StatusNoContent {
```

Comment:
did a version get released with the other status code? should this be `res.StatusCode != http.StatusNoContent && res.StatusCode != http.StatusOK`?

Reply:
you are right. I didn't think about that when I reverted. Let me throw up another :facepalm: